resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js

```diff
@@ -46,7 +46,7 @@
 	 * @returns {jQuery} Returns a jQuery promise
 	 * @see https://developers.facebook.com/docs/javascript/quickstart/v2.2
 	 */
-	$.loadFacebookAPI = function (callback) {
+	$.loadFacebookSDK = function (callback) {
```

Comment:
@lizlux I think the problem with adding it there is that `$.loadFacebookSDK` is only called with a callback function in side of FacebookClient.XFBML.js. Everywhere else there's no callback. Maybe I could add an additional `$('.sso-login').removeClass('hidden')` [here](https://github.com/Wikia/app/blob/SOC-261-2/resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js#L62)?